### PR TITLE
Remove acronym descriptions from Glossary headers

### PR DESCRIPTION
**Pull request by carolstran**
### Description

Clean up based on a discussion with @techwriter-dev on Slack where he noted:

> In an older PR related to the Glossary you once asked in a conversation about whether or not to include the expanded meaning of acronyms in the section heading.
>
> I am against the current practice of including it for a couple of reasons:
> 1. Internal anchors are way too long: `[IMEI](#imei---international-mobile-equipment-identity)`
> 2. It clutters the view and makes a quick skimming of the text body (or page contents pane) more difficult.
>
> Also, I noticed that en-dashes (maybe even em-dashes?) are sometimes used as hyphens.

🧹 🧹 🧹 

Screenshot 2023-03-30 at 10 46 02

### Additional context

Internal ticket 🎫  [SDOCS-334](https://emnify.atlassian.net/browse/SDOCS-334)

[SDOCS-334]: https://emnify.atlassian.net/browse/SDOCS-334?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -206,16 +251,22 @@ An IP address accessible from the public Internet.
...
 ## Service profile  
 A profile that defines the services and functionality of a device managed through the emnify platform.
 
-## SIM - Subscriber identification module
+## SIM
+_Subscriber Identification Module_
+
 A subscriber identification module (SIM) contains an integrated circuit ([IC](#ic---integrated-circuit)) that is often mounted on a plastic card.
 Pluggable SIMs mounted on plastic cards are offered in various form factors.
 A SIM stores data used to identify a subscriber ([IMSI](#imsi---international-mobile-subscriber-identity)) along with other network information for connecting and authenticating with a [mobile network operator (MNO)](https://www.emnify.com/iot-glossary/mno).
```
Comment (techwriter-dev) [inline docs/glossary.md:272]:
```suggestion
A SIM stores data used to identify a subscriber ([IMSI](#imsi)) along with other network information for connecting and authenticating with a [mobile network operator (MNO)](https://www.emnify.com/iot-glossary/mno).
```

File: docs/glossary.md [hunk 2]
```diff
@@ -86,13 +102,16 @@ A representation of the device which has a SIM installed.
...
 Because of the "e" (for *embedded*) in its name, *eSIM* is sometimes incorrectly used for referring to the MFF2 physical form factor of an [eUICC](#euicc---embedded-universal-integrated-circuit-card) chip that is designed to be permanently surface-mounted inside a device.
 Within the IoT industry, *eSIM* refers to the entire solution that is comprised of an eUICC-equipped [SIM](#sim---subscriber-identification-module) along with the software platform for [OTA provisioning](#ota-provisioning).
 Although eSIMs can be embedded directly in a device, they are also manufactured as pluggable SIM cards.
 The [emnify eSIM](/services/global-iot-sim) has capabilities not available with other eSIMs.
 
-## eUICC - Embedded universal integrated circuit card  
+## eUICC
+_Embedded Universal Integrated Circuit Card_  
 The embedded universal integrated circuit card (eUICC) is a component of a [SIM](#sim---subscriber-identification-module) card.
 It allows consumers and IoT manufacturers to provision the SIM with a new [operator profile](https://www.emnify.com/iot-glossary/mno) [over-the-air](#ota---over–the–air).
```
Comment (techwriter-dev) [inline docs/glossary.md:116]:
```suggestion
It allows consumers and IoT manufacturers to provision the SIM with a new [operator profile](https://www.emnify.com/iot-glossary/mno) [over-the-air](#ota).
```

**Review by techwriter-dev — CHANGES_REQUESTED** [review]
Added suggestions for fixing the broken links in `glossary.md`

Wasn't able to add a suggestion for line 167:
```

Following the introduction of [eUICC](#euicc---embedded-universal-integrated-circuit-card) SIM cards in 2021, there are situations where the ICCID can no longer be unique. 
```

**Comment by carolstran** [comment]
Excellent catch thank you so much @techwriter-dev 🙏🏼  updated and re-requested a review!

**Comment by carolstran** [comment]
Applied all requested changes and overriding the merge permissions since John is on vacation this week 😇

**Pull request merged**